src/Compilers/Core/Portable/DiagnosticAnalyzer/CompilationWithAnalyzers.cs

```diff
@@ -137,13 +137,6 @@ private CompilationWithAnalyzers(Compilation compilation, ImmutableArray<Diagnos
             _executingCompilationOrNonConcurrentTreeTask = null;
         }
 
-        private void AddExceptionDiagnostic(Exception exception, DiagnosticAnalyzer analyzer, Diagnostic diagnostic)
```

Comment:
So this seems to have uncovered a regression. This method should be hooked up in the constructor of CompilationWithAnalyzers and invoked when there is an exception from any analyzer. It seems this got regressed when we moved to AnalyzerDriver v2 in Dev14 Update3. Can you please retain this method and file a separate tracking issue to fix this and assign it to me?